easybuild/easyconfigs/a/Aspera-CLI/Aspera-CLI-3.7.2.354.010c3b8.eb

```diff
@@ -0,0 +1,40 @@
+# This is an easyconfig file for EasyBuild, see https://github.com/hpcugent/easybuild
+# Author: Daniel D. Kinnamon
+# Division of Human Genetics
+# The Ohio State University Wexner Medical Center
+
+easyblock = 'Tarball'
+
+name = 'Aspera-CLI'
+version = '3.7.2'
+versionsuffix = '.354.010c3b8'
+
+homepage = 'http://asperasoft.com'
+docurls = ['http://downloads.asperasoft.com/en/documentation/62']
+description = """IBM Aspera Command-Line Interface (the Aspera CLI) is
+a collection of Aspera tools for performing high-speed, secure data
+transfers from the command line. The Aspera CLI is for users and
+organizations who want to automate their transfer workflows."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+# Aspera CLI install script includes tarball inline and installs to a
+# fixed location. Need to include custom extract command to pull
+# tarball out of install script and pipe to tar. %s in the extraction
+# command will be replaced by the source filename.
+sources = [
+    ('%(namelower)s-%(version)s%(versionsuffix)s-linux-64-release.sh',
+     "sed '1,/^__ARCHIVE_FOLLOWS__$/d' %s |" +
+     'tar -xzpo --strip-components 1 -f -')
```

Comment:
@boegel: Will modify. I was trying not to exceed the sacred 80-character column width, but I agree that the single line is clearer.